docs/manual/devel_documentation.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: default
+title: rpm.org - Writing Documentation
+---
+# Writing Documentation
+
+There is a lot of 3rd party documentation on RPM and RPM
+packaging. We collect it on the [Documentation
+page](https://rpm.org/documentation.html) on rpm.org. Many of those
+are unfortunately not quite up-to-date.
+
+Get to an current and continuously updated documentation we are
+currently working on an in-tree Reference Manual. While still being less
+complete than some 3rd party documents we can enforce updates for all
+relevant changes and new features.
+
+## Reference Manual
+
+The [Reference
+Manual](https://rpm-software-management.github.io/rpm/manual/) is
+supposed to cover all user facing aspects of RPM but has still a long
+way to go.
+
+The markdown sources of the Reference Manual are found in
+`docs/manual`. They are not build into html or any other format by the
+make file. Instead the `master` branch is turned in to GitHub
```

Comment:
Cosmetic, but how about calling it Makefile instead? It's probably a bit more commonly used that way.